app/assets/javascripts/notices.js

```diff
@@ -0,0 +1,26 @@
+function setup_all_notice_writing_preview () {
```

Comment:
Identifier 'setup_all_notice_writing_preview' is not in camel case.<br>'setup_all_notice_writing_preview' is defined but never used.